### ISSUE TITLE
click and hold feature

### ISSUE THREAD

**Issue by ericavijay** [wenzhixin/bootstrap-table#3119]
Is there a way to update bootstrap table to have a "click and hold" feature on row that is different from the click action on the row.  For example support perhaps drag and drop on click-and-hold instead of doing the default onclick action.

Thanks
Vijay

**Comment by djhvscf** [comment]
This is not a current feature

**Comment by ericavijay** [comment]
Yes! correct this is not a bug. Sorry I didn't add feature label 
Thanks

**Comment by djhvscf** [comment]
@ericavijay if you can create a **PR** please see this SO [answer](http://stackoverflow.com/questions/4080497/how-can-i-listen-for-a-click-and-hold-in-jquery/4081293#4081293)
